packages/fast-components-react-base/src/utilities/index.ts

```diff
@@ -13,7 +13,8 @@ declare global {
     }
 }
 
-export { DisplayNamePrefix };
+export { DisplayNamePrefix, WindowWithIntersectionObserver, WindowWithResizeObserver };
```

Comment:
since you are wholesale exporting this it will include the export of `DisplayNamePrefix` which I don't think we want, that's more of an private variable just for the package.

Reply:
Yep, good catch.